src/pretix/base/models/organizer.py

```diff
@@ -464,3 +465,25 @@ def get_events_with_permission(self, permission, request=None):
             return self.get_events_with_any_permission()
         else:
             return self.team.organizer.events.none()
+
+
+class OrganizerFooterLink(models.Model):
+    """
+    A footer link assigned to an organizer.
+    """
+    organizer = models.ForeignKey('Organizer', on_delete=models.CASCADE, related_name='footer_links')
```

Comment:
How about adding a sort-field? Currently to change the order, I need to delete and add.

Reply:
We can always add that in later, my time budget did not allow for it now and I don't think it's critically important here. I don't want to encourage people adding loads of links here anyawys.